src/attributes.jl

```diff
@@ -1260,8 +1273,7 @@ A constraint attribute for the `BasisStatusCode` of some constraint in result
 
 See [`ResultCount`](@ref) for information on how the results are ordered.
 
-**For the basis status of a variable, query the corresponding `SingleVariable`
-constraint that enforces the variable's bounds.**
+For the basis status of a variable, query [`VariableBasisStatus`](@ref).
```

Comment:
Maybe also say explicitly that `ConstraintBasisStatus` for `SingleVariable` is not defined: solver should not implement it and user should not call it.